packages/mds-agency/handlers/create-event.ts

```diff
@@ -111,7 +111,7 @@ export const createEventHandler = async (
     ...req.body,
     device_id,
     provider_id,
-    telemetry: { ...req.body.telemetry, provider_id },
+    telemetry: { ...req.body.telemetry, provider_id, recorded },
```

Comment:
Why do we need to do this now?

Reply:
it's part of the schema for telemetry. It's not my fault.